<launch>
<node name="systemControlNode" pkg="reconfigure" type="reconfigure_node" />
</launch>
